HLTrigger/JSONMonitoring/plugins/TriggerJSONMonitoring.cc

```diff
@@ -529,7 +531,14 @@ TriggerJSONMonitoring::endLuminosityBlockSummary(const edm::LuminosityBlock& iLu
 void
 TriggerJSONMonitoring::globalEndLuminosityBlockSummary(const edm::LuminosityBlock& iLumi, const edm::EventSetup& iSetup, const LuminosityBlockContext* iContext, hltJson::lumiVars* iSummary)
 {
-  if (iSummary->processed->value().at(0)!=0) {
+
+  unsigned int iLs  = iLumi.luminosityBlock();
+  unsigned int iRun = iLumi.run();
+
+  evf::FastMonitoringService * fms_ = (evf::FastMonitoringService *)(edm::Service<evf::MicroStateService>().operator->());
+  bool writeFiles = fms_ ? fms_->getEventsProcessedForLumi(iLs)>0 : true;
```

Comment:
Ops, apologies, ==fms_== is used only here, and there is a check - fooled by the name, I though it was a member available elsewhere.